example-app/src/main/res/values/strings.xml

```diff
@@ -8,6 +8,10 @@
     <string name="components_title">Components</string>
     <string name="card_component_entry">Card</string>
 
+    <string name="bacs_component_entry">BACS (using bottom sheet)</string>
+    <string name="bacs_country_error">Your country code must be GB</string>
```

Comment:
Can we change this string and blik to take the country code as a parameter? Same for the other string and currency code